### DISCUSSION TITLE
Best font for vs code editor in dark

### DISCUSSION THREAD

**Discussion by J-Nokwal** [Schweinepriester/github-profile-achievements#497] [Q&A] [answered]
Best font for vs code editor in dark

**Comment by bawejagb** [comment, marked as answer]
I recommend you use the **JellyFish Theme** Extention, it has very great text fonts. 
Must Try, I've been using it for the past few months, and I love it.
Link: [https://marketplace.visualstudio.com/items?itemName=PawelBorkar.jellyfish](url)
